polkadot/roadmap/implementers-guide/src/protocol-approval.md

```diff
@@ -296,6 +296,16 @@ provide somewhat more security.
 
 TODO: When?  Is this optimal for the network?  etc.
 
+## Approval coalescing
+To reduce the necessary network bandwidth and cpu time when a validator has more than one candidate to approve we are
+doing our best effort to send a single message that approves all available candidates with a single signature.
+The implemented heuristic, is that each time we are ready to create a signature and send a vote for a candidate we
+delay sending it until one of three things happen:
+- We gathered a maximum of `MAX_APPROVAL_COALESCE_COUNT` candidates that we are ready to vote for.
+- `MAX_APPROVAL_COALESCE_WAIT_TICKS` have passed since the we were ready to approve the candidate.
```

Comment:
```suggestion
- `MAX_APPROVAL_COALESCE_WAIT_TICKS` have passed since checking oldest candidate and we were ready to sign and send the approval message
```